### PR TITLE
fix: upgrade python openapiclient version, fixes #9770

### LINKED ISSUE #9770: Python SDK list_workflows break when Workflow in progress
### Pre-requisites

- [X] I have double-checked my configuration
- [X] I can confirm the issues exists when I tested with `:latest`
- [ ] I'd like to contribute the fix myself (see [contributing guide](https://github.com/argoproj/argo-workflows/blob/master/docs/CONTRIBUTING.md))

### What happened/what you expected to happen?

I expect the list_workflows method in the Python SDK to return the result successfully. 

If you try to list workflows via the Python SDK while there is a workflow in progress, you will get an error.

```
import time
import argo_workflows
from argo_workflows.api import workflow_service_api
with argo_workflows.ApiClient(self.configuration) as api_client:
            api_instance = workflow_service_api.WorkflowServiceApi(api_client)
            api_response = api_instance.list_workflows(self._NAMESPACE, _check_return_type=False)
            print(api_response)
```

```
  File "/Users/rajaiealkorani/repos/crawl/venv/lib/python3.9/site-packages/argo_workflows/model_utils.py", line 512, in to_dict
    return model_to_dict(self, serialize=False)
  File "/Users/rajaiealkorani/repos/crawl/venv/lib/python3.9/site-packages/argo_workflows/model_utils.py", line 1663, in model_to_dict
    res.append(model_to_dict(v, serialize=serialize))
  File "/Users/rajaiealkorani/repos/crawl/venv/lib/python3.9/site-packages/argo_workflows/model_utils.py", line 1635, in model_to_dict
    if model_instance._composed_schemas:
AttributeError: 'dict' object has no attribute '_composed_schemas'
```

This error doesn't show up if there are 0 workflows in the request namespace.

### Version

v6.4.1

### Paste a small workflow that reproduces the issue. We must be able to run the workflow; don't enter a workflows that uses private images.

```YAML
the presence of any workflows (running, failed, completed, etc...) causes this issue
```

### Logs from the workflow controller

not relevant

### Logs from in your workflow's wait container

 not relevant

### PR DESCRIPTION
**Pull request by tczhao**
Fixes #9770

related commit in openapi client cli
https://github.com/OpenAPITools/openapi-generator/pull/11234

---

Had attempted to align the openapi version for java to the same as python
but meeting errors.
I'm extremely raw in java, would take quite some time to understand what's happening.
Maybe we can create an issue for that once this merged

---

Please do not open a pull request until you have checked ALL of these:

* [x] Create the PR as draft .
* [x] Run `make pre-commit -B` to fix codegen and lint problems.
* [x] Sign-off your commits (otherwise the DCO check will fail).
* [x] Use [a conventional commit message](https://www.conventionalcommits.org/en/v1.0.0/) (otherwise the commit message check will fail).
* [x] "Fixes #" is in both the PR title (for release notes) and this description (to automatically link and close the issue).
* [x] Add unit or e2e tests. Say how you tested your changes. If you changed the UI, attach screenshots.
* [x] Github checks are green.
* [x] Once required tests have passed, mark your PR "Ready for review".

If changes were requested, and you've made them, dismiss the review to get it reviewed again.

### CODE REVIEW

File: .github/workflows/ci-build.yaml [hunk 1]
```diff
@@ -173,7 +173,9 @@ jobs:
           sudo find /usr/local/include -type f | xargs sudo chmod a+r
           sudo find /usr/local/include -type d | xargs sudo chmod a+rx
           ls /usr/local/include/google/protobuf/
-      - run: docker pull openapitools/openapi-generator-cli:v5.2.1 &
+      - run: |
+          docker pull openapitools/openapi-generator-cli:v5.4.0 &
+          docker pull openapitools/openapi-generator-cli:v5.2.1 &
```
Comment (terrytangyuan) [inline .github/workflows/ci-build.yaml:178]: Why are we pulling two versions?

**Review by terrytangyuan — COMMENTED** [review]
We may had a hard requirement on lower versions of the generator to work around a particular issue. 

@flaviuvadan Could you help double check?

File: .github/workflows/ci-build.yaml (hunk 1 above)
Comment (tczhao) [reply .github/workflows/ci-build.yaml:178]: The python gen is upgraded to v5.4.0
Java gen is facing some issues in v5.4.0, unchanged for now, remain at 5.2.1

**Comment by alexec** [comment]
The Java version is having issues with `Time`.

**Review by alexec — APPROVED** [review]

**Review by flaviuvadan — APPROVED** [review]

**Pull request merged**